README.md

```diff
@@ -1,150 +1,119 @@
-    ~~~
-
-    before run the script by
-
-    ~~~
-    $ ./swc-workshop-create
-    ~~~
-
-4.  Run the script with those parameters.
-
-5.  Go into your newly-created repository.
-
-6.  Edit `index.html`.  Hints are embedded in the file, and full
-    instructions are in [CUSTOMIZATION.md](CUSTOMIZATION.md).
-
-7.  If you have installed the software described below:
-
-    1.  Check your changes by running `tools/check.py` inside your
-        repository.
-
-    2.  Preview your changes by running `tools/preview` and looking at
-        `_site/index.html`.
-
-    3.  For some links work properly, **including the one to Eventbrite**, you
-        will need to look at `_site/index.html` provide by a HTTP server that
-        can be done using
-
-        ~~~
-        $ jekyll server -d _site
-        ~~~
-
-        and look at http://localhost:4000.
+4.  Select the owner for your new repository.
+    (This will probably be you, but may instead be an organization you belong to.)
 
-8.  Commit your changes and push to the `gh-pages` branch of your
-    repository.
+5.  Choose a name for your workshop website repository.
+    This name should have the form `YYYY-MM-DD-site`,
+    e.g., `2015-07-01-miskatonic`.
 
-9.  Send the workshop coordinators the URL for your GitHub repository.
+6.  Make sure the repository is public.
 
-If the identifier for your workshop is `2015-07-01-esu`, and your
-GitHub username is `ghopper`, your workshop repository will be
-`https://github.com/ghopper/2015-07-01-esu` and the website for your
-workshop will be `https://ghopper.github.io/2015-07-01-esu`.  (Note:
-the workshop coordinators will want the former URL, not the latter.)
+7.  At this point, you should have a page like this:
 
-## Manual Setup
+    ![](http://software-carpentry.org/img/workshop-template/using-github-import.png)
 
-You can set up your repository manually instead of using the automated
-`create` script.  As above, we will assume that your user ID is
-`ghopper` and the identifier for your workshop is `2015-07-01-esu`.
+    You can now click "Begin Import".
+    When the process is done,
+    you can click "Continue to repository" to visit your newly-created repository.
 
-1.  Create an empty repository on GitHub called `2015-07-01-esu`.
+## Customizing Your Website
 
-2.  Clone the template repository to your computer in a directory with
-    the same name as your workshop identifier:
-
-    ~~~
-    $ git clone -b gh-pages -o upstream https://github.com/swcarpentry/workshop-template.git 2015-07-01-esu
-    ~~~
-
-3.  Go into that directory using
-
-    ~~~
-    $ cd 2015-07-01-esu
-    ~~~
+1.  Go into your newly-created repository,
+    which will be at `http://github.com/your_username/YYYY-MM-DD-site`.
+    For example,
+    if `your_username` is `gvwilson`,
+    the repository's URL will be `https://github.com/gvwilson/2015-07-01-mistaktonic`.
 
-4.  Add your GitHub repository as a remote called `origin` using
+2.  Edit `index.html`.
```

Comment:
Might be helpful to state that the user should edit the first block of code in index.html to personalize the workshop instructors, location, etc.

I like the image with the pencil circled.